oarepo_runtime/services/facets/params.py

```diff
@@ -55,10 +55,17 @@ def facet_groups(self):
         return None
 
     def identity_facets(self, identity: Identity):
+        global_search_model = False
+        for model in current_app.config.get("GLOBAL_SEARCH_MODELS", []):
+            service_config = obj_or_import_string(model["service_config"])
+            if service_config == self.config:
```

Comment:
we need to profile this sometime to see if the obj_or_import_string is a fast operation as it is done on every request